Bottom: 35fa13cd8f52c2f6da7b2b5f26bcb1e2c060e568
Top:    19286476a36c5ce8b386d705d80ac0437ce5eead
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-07-04 04:24:33 +0100

Remove function SearchScreenGridSlop() and use SearchObjectByLocation() directly


---

diff --git a/src/crosshair.c b/src/crosshair.c
index 1a81c25..83734d1 100644
--- a/src/crosshair.c
+++ b/src/crosshair.c
@@ -1108,9 +1108,9 @@ FitCrosshairIntoGrid (Coord X, Coord Y)
       if (TEST_FLAG (RUBBERBANDFLAG, PCB))
         {
 
-          ans = SearchScreenGridSlop (Crosshair.X, Crosshair.Y,
-                                      LINE_TYPE,
-                                      &ptr1, &ptr2, &ptr3);
+          ans = SearchObjectByLocation (LINE_TYPE,
+                                        &ptr1, &ptr2, &ptr3,
+                                        Crosshair.X, Crosshair.Y, PCB->Grid / 2);
           if (ans != NO_TYPE) {
             double angle;
             int octant;
